LearningMissionSimulation/PlayGroundGavril.cs

```diff
@@ -1,14 +1,58 @@
-﻿using System;
+﻿using LearningMissionLab;
+using System;
 using System.Collections.Generic;
 using System.Text;
 
 namespace LearningMissionSimulation
 {
-    public  class PlayGroundGavril
+    public class PlayGroundGavril
     {
-        public PlayGroundGavril()
+        List<Account> pendingAccountList = new List<Account>();
+        List<Student> studentList = new List<Student>();
+        List<Instructor> instructorList = new List<Instructor>();
+        Dictionary<Guid, Account> accountDictionary = new Dictionary<Guid, Account>();
+        public void CreateAccounts(int accountCount)
         {
+            for (int i = 0; i < accountCount; i++)
+            {
+                Account account = ObjectGenerator.GenerateAccount();
 
+                accountDictionary.Add(account.Id, account);
+                if (account.Role == Role.Student)
+                {
+                    Student student = ObjectGenerator.GenerateStudent(account.Id);
+                    studentList.Add(student);
+
+                    if (student.DateOfBirth == new DateTime(8, 9, 1981))
+                    {
+                        pendingAccountList.Add(account);
+                    }
+                }
+                else if (account.Role == Role.Instructor)
+                {
+                    Instructor instructor = (ObjectGenerator.GenerateInstructor(account.Id));
+                    instructorList.Add(instructor);
+                }
+                if ((account.Role == Role.Student || account.Role == Role.Instructor) && account.Status == Status.Pending)
```

Comment:
Lines 36-40 are redundant, you already added pending student accounts to the list, you have to do the same for instructors. I know that you copy paste Garush's implementation. But it was not correct and was not final, he fixed his PR after that. It is already in the master branch. Please check it out and make your fixes accordingly.